pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -175,8 +178,10 @@ func (rc *reconciler) reconcile() {
 			err = rc.attacherDetacher.DetachVolume(attachedVolume.AttachedVolume, verifySafeToDetach, rc.actualStateOfWorld)
 			if err == nil {
 				if !timeout {
+					// TODO-P1: I think we don't expect this code path to be hit... so Errorf / Warningf?
```

Comment:
It is the normal detach operation path. It passes the safe check to detach and it is not caused by timeout.